tests/compiletest.rs

```diff
@@ -465,8 +121,7 @@ fn main() {
     // Panic tests expect backtraces to be printed.
     env::set_var("RUST_BACKTRACE", "1");
 
-    let target = get_target();
-    ui(Mode::Pass, "tests/run-pass", &target);
-    ui(Mode::Panic, "tests/run-fail", &target);
-    ui(Mode::UB, "tests/compile-fail", &target);
+    ui(Mode::Pass, "tests/run-pass");
+    ui(Mode::Panic, "tests/run-fail");
+    ui(Mode::UB, "tests/compile-fail");
```

Comment:
Looking at the names of the folders, maybe we should call the variants `RunFail` and `CompileFail`?

Though "compile-fail" is not  a great name and we should probably rename the folder instead...

Reply:
> I didn't want to cause the folder renaming churn.

Fair, we can leave that to a separate PR.

> I want run-pass, run-panic and run-fail

The "run" part looks rather redundant in these names...

Reply:
yeaaa... we don't really have compile-fail tests, so leaving out the run part makes sense